src/api/app/queries/notifications_finder.rb

```diff
@@ -44,4 +44,10 @@ def for_notifiable_type(type = 'unread')
   def for_project_name(project_name)
     unread.joins(:projects).where(projects: { name: project_name })
   end
+
+  def current_notification(notification_id)
```

Comment:
`current_notification` is a bit too generic, making this method somewhat cryptic.

How about `for_subscribed_user_by_id`? It's longer, but much clearer. It would read like `NotificationsFinder.new.for_subscribed_user_by_id(12345)`.
```suggestion
  def for_subscribed_user_by_id(notification_id)
```

Reply:
Makes sense, since it could be used not only for the case when it is required for the current notification. Your name is more general and reflects of what it does inside. Thank you.